bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/configurable/ConfigConstants.java

```diff
@@ -25,10 +25,6 @@
  */
 public class ConfigConstants {
 
-    public static final String CONFIG_FILE_NAME = "Config.toml";
-
-    public static final String CONFIG_ENV_VARIABLE = "BALCONFIGFILE";
-
     public static final String CONFIG_SECRET_ENV_VARIABLE = "BALSECRETFILE";
```

Comment:
this can also be removed (lets do in next pr)

Reply:
I think this might be removed with the PR for removing encryption. @warunalakshitha Are we removing this?

Reply:
Yes I will remove this